plenum/server/message_req_processor.py

```diff
@@ -0,0 +1,259 @@
+from typing import Dict
+from typing import List
+
+from plenum.common.constants import LEDGER_STATUS, PREPREPARE, CONSISTENCY_PROOF, \
+    PROPAGATE
+from plenum.common.messages.fields import RequestIdentifierField
+from plenum.common.messages.node_messages import MessageReq, MessageRep, \
+    LedgerStatus, PrePrepare, ConsistencyProof, Propagate
+from plenum.common.request import Request
+from plenum.common.types import f
+from plenum.server import replica
+from stp_core.common.log import getlogger
+
+
+logger = getlogger()
+
+
+class MessageReqProcessor:
```

Comment:
There are tests present in `plenum/test/node_request/message_request/test_valid_message_request.py`